include/geom/face_quad4.h

```diff
@@ -159,6 +159,27 @@ class Quad4 : public Quad
 
 };
 
+class QuadShell4 : public Quad4
```

Comment:
Agreed with @jwpeterson on this one.

Although "number of new lines of code" is good to keep an eye on too.  In hindsight there are a whole slew of methods that I should have put into an ElemTools:: namespace rather than the Elem class...